### PR TITLE
Update eventing-kafka components to 1.2

### PR DESCRIPTION
**Pull request by aliok**
Update eventing-kafka components to 1.2 - which actually is only the KafkaChannel

Going to swap the KafkaChannel later.

### CODE REVIEW

**Comment by skonto** [comment]
I guess images are not built yet.

**Comment by matzew** [comment]
@aliok you need to rebase

**Comment by aliok** [comment]
/retest

Images are now there: https://github.com/openshift/release/pull/27653

**Comment by aliok** [comment]
/test 4.10-upstream-e2e-aws-ocp-410

Let me retry even though it looks like there's a real problem.

**Comment by aliok** [comment]
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-knative_serverless-operator/1473/pull-ci-openshift-knative-serverless-operator-main-4.10-upstream-e2e-aws-ocp-410/1512297152271880192/artifacts/upstream-e2e-aws-ocp-410/serverless-e2e/artifacts/build-mtqIVcKq/knative-eventing-logs/knative-eventing/eventing-controller-7679bc8cb7-b98dj-eventing-controller.log
```
{"level":"error","ts":"2022-04-08T05:27:30.793Z","logger":"controller","caller":"duck/controller.go:55","msg":"Error getting source informer","knative.dev/pod":"eventing-controller-7679bc8cb7-b98dj","GVR":"sources.knative.dev/v1beta1, Resource=kafkasources","error":"the server could not find the requested resource","stacktrace":"knative.dev/eventing/pkg/reconciler/source/duck.NewController.func1\n\t/go/src/knative.dev/eventing/pkg/reconciler/source/duck/controller.go:55\nknative.dev/eventing/pkg/reconciler/source/crd.
```

**Comment by matzew** [comment]
@aliok not sure, but perhaps my bump for eventing core to 1.2.1 is _related_ ?

File: knative-operator/hack/update-manifests.sh [hunk 1]
```diff
@@ -72,7 +72,6 @@ git apply "$root/knative-operator/hack/003-broker-remove-duplicated-crds.patch"
 git apply "$root/knative-operator/hack/004-remove_new_channel_cfg.patch"
 
 # Fix for SRVKE-1171
```
Comment (matzew) [inline knative-operator/hack/update-manifests.sh:74]: let's remove that comment as well, since that was for the patch

**Comment by matzew** [comment]
So far looks good

**Comment by aliok** [comment]
> @aliok not sure, but perhaps my bump for eventing core to 1.2.1 is related ?

Could be. Let me wait for https://github.com/openshift-knative/serverless-operator/pull/1482 to get merged

**Comment by matzew** [comment]
@aliok this needs a rebase now

**Comment by aliok** [comment]
/hold

This needs to hold until https://github.com/openshift-knative/serverless-operator/pull/1482 is reverted with https://github.com/openshift-knative/serverless-operator/pull/1488 and then properly merged again.

**Comment by matzew** [comment]
/close

this is replaced by: https://github.com/openshift-knative/serverless-operator/pull/1497

**Pull request closed without merging**